Artifacts/windows-certificate-from-keyvault/InstallCertificate.ps1

```diff
@@ -62,8 +62,8 @@ try{
     Import-Module AzureRm
 
     $azureAccountName = $azureServicePrincipalClientId
-    $azurePassword = ConvertTo-SecureString $azureServicePrincipalKey -AsPlainText -Force
-    $psCred = New-Object System.Management.Automation.PSCredential($azureAccountName, $azurePassword)
+    #$azurePassword = ConvertTo-SecureString $azureServicePrincipalKey -AsPlainText -Force
```

Comment:
Fixed in https://github.com/Azure/azure-devtestlab/pull/248/commits/42f85103f9c4e29f4e1c16c767eeeed8689bcdf2